src/state/client/ClientState.js

```diff
@@ -32,6 +32,19 @@ const ClientState = props => {
     });
   };
 
+  // UPDATE CLIENT INFO
+  const updateProfile = async update => {
+    const res = await apiAuth(authState).put(
+      `https://labspt12-express-groomer-c-api.herokuapp.com/profiles`,
```

Comment:
Can we replace the URI with the env var REACT_APP_API_URI?

Something like `'${process.env.REACT_APP_API_URI}/profile'`